### ISSUE TITLE
Support GitLab CI

### ISSUE THREAD

**Issue by chauffer** [siddharthkp/bundlesize#19]
https://docs.gitlab.com/ce/ci/

**Comment by siddharthkp** [comment]
@chauffer, I have not used gitlab, can you guide me towards their status APIs?

From what I understand, gitlab has it's own CI, which will come in handy.

**Comment by chauffer** [comment]
@siddharthkp for the most part, it's simply Docker containers, where exit code != 0 is failure.

Some env vars are available, fwiw: https://docs.gitlab.com/ce/ci/variables/README.html

What other information would you need? Happy to jump on gitter or something to help out :D

**Comment by siddharthkp** [comment]
Thanks, this helps.

I'm not sure what's the best way to test this 🤔 

I'll create a fork on gitlab?

**Comment by chauffer** [comment]
yep, probably good enough as gitlab.com has free shared runners :D

**Comment by jagzviruz** [comment]
@siddharthkp you can try setting up [repository mirroring](https://docs.gitlab.com/ee/workflow/repository_mirroring.html) on Gitlab to avoid maintaining two versions.

**Comment by blittle** [comment]
What's the status of this? Would it be worthwhile for me to jump in and try and implement it?

**Comment by SilentImp** [comment]
Any updates?
